Source/WebCore/css/CSSFontFaceSrcValue.h

```diff
@@ -59,9 +60,54 @@ class CSSFontFaceSrcLocalValue final : public CSSValue {
     WeakPtr<SVGFontFaceElement, WeakPtrImplWithEventTargetData> m_element;
 };
 
+enum class FontTech : uint8_t {
```

Comment:
Is this the right file to define this enum? I think there is a better file but I can’t remember off the top of my head what its name is.